ci-jobs/crowdin-sync-translations.js

```diff
@@ -1,72 +1,54 @@
-const { logger, tempDir, fs, zip } = require('@appium/support');
-const request = require('request');
+const { tempDir, fs, zip, net } = require('@appium/support');
 const path = require('path');
-const { createWriteStream } = require('fs');
-const B = require('bluebird');
-const { asyncify } = require('asyncbox');
+const {
+  log, RESOURCES_ROOT, ORIGINAL_LANGUAGE, performApiRequest
+} = require('./crowdin-common');
+const { waitForCondition } = require('asyncbox');
 
-const log = logger.getLogger('CROWDIN');
+const BUILD_TIMEOUT_MS = 1000 * 60 * 10;
+const BUILD_STATUS = {
+  finished: 'finished',
+  created: 'created',
+  inProgress: 'inProgress',
+  canceled: 'canceled',
+  failed: 'failed',
+};
 
-const PROJECT_ID = process.env.CROWDIN_PROJECT_ID;
-const PROJECT_KEY = process.env.CROWDIN_PROJECT_KEY;
-if (!PROJECT_ID || !PROJECT_KEY) {
-  throw new Error(`Both CROWDIN_PROJECT_ID and CROWDIN_PROJECT_KEY environment ` +
-    `variables must be set`);
-}
-const RESOURCES_ROOT = path.resolve('assets', 'locales');
-const ORIGINAL_LANGUAGE = 'en';
-const USER_AGENT = 'Appium Desktop';
-
-async function exportTranslations () {
-  const options = {
-    url: `https://api.crowdin.com/api/project/${PROJECT_ID}/export?key=${PROJECT_KEY}`,
-    port: 443,
-    method: 'GET',
-    headers: {
-      'User-Agent': USER_AGENT,
-    },
-  };
-  return await new B((resolve, reject) => {
-    request(options)
-      .on('error', reject)
-      .on('response', (res) => {
-        if (res.statusCode >= 400) {
-          return reject(`Cannot export the translated resources in Crowdin. Error code: ${res.statusCode}`);
-        }
-        log.info(`Successfully exported Crowdin translations`);
-      })
-      .on('close', resolve);
+async function buildTranslations () {
+  log.info('Building project translations');
+  const {data: buildData} = await performApiRequest('/translations/builds', {
+    method: 'POST',
```

Comment:
The Crowdin documentation doesn't say whether `skipUntranslatedStrings`, `skipUntranslatedFiles` and `exportApprovedOnly` are set to `false` by default, but I assume they are.

Reply:
as far as I can see they are